src/core/session.py

```diff
@@ -208,6 +208,20 @@ def init(self, directory: str, load: bool = True) -> None:
         # read project
         if load:
             QgsProject.instance().read(self.db.projectUri)
+            for layer in [
+                self._gpsLayer,
+                self._worldLayer,
+                self._speciesLayer,
+                self._followersLayer,
+                self._observersLayer,
+                self._sightingsLayer,
+                self._environmentLayer,
+                self._surveyLayer,
+                self._transectLayer,
+                self._strateLayer,
+                self._plateformLayer,
+            ]:
+                layer._init(layer.layer)
```

Comment:
NB : This is necessary to update old project

Reply:
Ok :+1:.

We'll probably need an update mechanism like in stormm-ground at one point (but it's not necessary for now).